core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -846,6 +848,14 @@ public void statusChanged(PlaybackServiceMediaPlayer.PSMPInfo newInfo) {
                     break;
             }
 
+            // Trigger an update to the Quick Settings tile (see QuickSettingsTileService.java)
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+                TileService.requestListeningState(
+                        getApplicationContext(),
+                        new ComponentName(getApplicationContext(), QuickSettingsTileService.class.getName())
+                );
+            }
```

Comment:
This can be more compact.

```suggestion
            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
                TileService.requestListeningState(getApplicationContext(),
                        new ComponentName(getApplicationContext(), QuickSettingsTileService.class));
            }
```

Reply:
Implemented